packages/main/src/CheckBox.hbs

```diff
@@ -1,20 +1,17 @@
-<div {{> controlData}}
+<div
 	class="{{classes.main}}"
-	style="{{styles.main}}"
 	role="checkbox"
-	aria-checked="{{ctr.checked}}"
-	?aria-readonly="{{ariaReadonly}}"
+	aria-checked="{{checked}}"
+	aria-readonly="{{ariaReadonly}}"
+	aria-disabled="{{ariaDisabled}}"
 	tabindex="{{tabIndex}}">
-		<div id="{{ctr._id}}-CbBg" class="{{classes.inner}}">
-			<input id="{{ctr._id}}-CB" type='checkbox' ?checked="{{ctr.checked}}" ?readonly="{{ctr.readonly}}" data-sap-no-tab-ref/>
+		<div id="{{_id}}-CbBg" class="{{classes.inner}}">
+			<input id="{{_id}}-CB" type='checkbox' ?checked="{{checked}}" ?readonly="{{readonly}}" ?disabled="{{disabled}}"data-sap-no-tab-ref/>
```

Comment:
There is no space after disabled, maybe it's ok like this too

Reply:
I added space just in case